azure-communication-ui/azure-communication-ui/src/main/java/com/azure/android/communication/ui/presentation/DependencyInjectionContainerHolder.kt

```diff
@@ -40,17 +44,21 @@ internal class DependencyInjectionContainerHolder(application: Application) :
         DependencyInjectionContainerImpl(application, instanceId)
     }
 
+    val localizationProvider: LocalizationProvider by lazy {
```

Comment:
we should not hold anything in the DependencyInjectionContainerHolder other then DependencyInjectionContainer.
val localizationProvider: LocalizationProvider needs to be moved to DependencyInjectionContainer, similar as appStore. Then we can use it from there:
SetupViewModelFactory(container.appStore, container.localizationProvider)

Reply:
Thanks Pavel for suggesting the flow.